tests/integration/test_integration.py

```diff
@@ -49,6 +50,9 @@ async def test_relate_and_wait_for_active_status(
         status="active",
         timeout=TIMEOUT,
     )
+    await ops_test.model.integrate(
+        relation1=f"{APP_NAME}:logging", relation2=GRAFANA_AGENT_CHARM_NAME
```

Comment:
After integrating with grafana agent we should likely check that the du charm status goes back to active/idle (with wait_for_idle)

Reply:
you are totally right